### ISSUE TITLE
DBus interface name does not match XML definition file

### ISSUE THREAD

**Issue by yan12125** [keepassxreboot/keepassxc#5355]
## Overview
In https://github.com/keepassxreboot/keepassxc/blob/develop/src/gui/org.keepassxc.KeePassXC.MainWindow.xml, the interface name is declared to be `org.keepassxc.KeePassXC.MainWindow`, while the actual interface name is `org.keepassxc.MainWindow` [1].

[1] https://github.com/keepassxreboot/keepassxc/blob/develop/src/gui/MainWindowAdaptor.h

## Steps to Reproduce
1. Try to open a database with the instruction on the Wiki [2]
```
qdbus org.keepassxc.KeePassXC.MainWindow /keepassxc org.keepassxc.KeePassXC.MainWindow.openDatabase /path/to/database.kdbx passwd /path/to/key
```
[2] https://github.com/keepassxreboot/keepassxc/wiki/Using-DBus-with-KeePassXC

## Expected Behavior
It works.

## Actual Behavior
The working command is different from what the Wiki says

```
qdbus org.keepassxc.KeePassXC.MainWindow /keepassxc org.keepassxc.MainWindow.openDatabase /path/to/database.kdbx passwd /path/to/key
```

In https://github.com/keepassxreboot/keepassxc/commit/98a89e4f2c973683aef8f57eb527d3b7110d6287#diff-29eaf0e2a89ffcb6e44f29774368a853, `org.keepassxc.MainWindow.xml` is renamed to `org.keepassxc.KeePassXC.MainWindow.xml`, and `MainWindowAdaptor.h` is regenerated, but from the comment it appears to be generated with the old interface name.
```
Command line was: qdbusxml2cpp -c MainWindowAdaptor -a MainWindowAdaptor.h:MainWindowAdaptor.cpp org.keepassxc.MainWindow.xml
```
After regenerating MainWindowAdaptor.{h,cpp} and rebuilding, the `openDatabase` command mentioned on Wiki works.

## Context

KeePassXC - Version 2.6.1
Revision: 9a35bba

Qt 5.15.0
Debugging mode is disabled.

Operating system: Arch Linux
CPU architecture: x86_64
Kernel: linux 5.8.5-arch1-1

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
 libgcrypt 1.8.6

Desktop Env: LXQt
Windowing System: X11

**Comment by droidmonkey** [comment]
So does the wiki just need to be updated?

**Comment by yan12125** [comment]
> So does the wiki just need to be updated?

Not really. Per https://github.com/keepassxreboot/keepassxc/pull/283#discussion_r163618277, I believe the intention is switching to longer `org.keepassxc.KeePassXC.MainWindow` interface name. This involves regenerating `MainWindowAdaptor.{h,cpp}` from `org.keepassxc.KeePassXC.MainWindow` using qdbusxml2cpp. The Wiki needs no update as it will match the code behavior after the qdbusxml2cpp step.

**Comment by droidmonkey** [comment]
@yan12125 did you want to merge your proposed change for this one?

**Comment by yan12125** [comment]
Sure, I can rebase and create a pull request this weekend.

**Comment by yan12125** [comment]
Somehow this issue is not closed automatically when the pull request is merged (as the PR targets release/2.7 instead of develop?). Closing manually and thanks for reviewing!

**Comment by droidmonkey** [comment]
Issues only auto-close on GitHub when the commit is merged to the default branch (develop in our case)... which is super annoying. Most annoying bug on GitHub tbh.
